src/components/Menu/styles.js

```diff
@@ -1,62 +1,110 @@
+//export const styles = {
+  export const logoWrap = mq({
     marginLeft: '0!important',
-    paddingTop: '24px!important',
+    paddingTop: '13px!important',
     paddingBottom: '0!important',
     paddingRight: '0!important',
-    paddingLeft: '0!important',
-  },
-  logoImage: {
-    width: 168.64,
-    height: 21.29,
-    marginBottom: 0,
+    paddingLeft: ['15px!important','15px!important','15px!important','10px!important'],
+    fontSize: ['1em','1.5em','2.1em','2.1em'],
+  })
+
+  export const logoLink = css ({
+    textDecoration: 'none!important',
+    color: 'black!important',
+    })
+
+    export const subMenuCol = mq({
+    '& > :nth-child(2)': {
+      paddingTop: ['40px','40','0','0'],
   },
-  subMenuCol: {
     float: 'left',
     display: 'block',
     position: 'relative',
     listStyle: 'none',
     margin: 0,
     paddingLeft: 0,
-  },
-  subMenuLink: {
+  })
+
+  export const shopLink = ({
+    float: 'right',
+    display: 'block',
+    position: 'relative',
+    listStyle: 'none',
+    margin: 0,
+    paddingLeft: 0,
+  })
+
+  export const subMenuLink = mq({
+    paddingBottom: '40px',
     cursor: 'pointer',
-    fontSize: '2em',
+    fontSize: ['1em','1.5em','2em','2em'],
+    color: '#CACAC8!important',
+  })
+
+  export const subMenuLinkBlack = mq({
+    cursor: 'pointer',
+    fontSize: ['1em','1.5em','2em','2em'],
     color: 'black!important',
-  },
-  subMenuLinkShop: {
+  })
+  
+  export const subMenuLinkShop = mq({
     color: '#CACAC8',
-    fontSize: '2em',
+    fontSize: ['1em','1.5em','2em','2em'],
     cursor: 'pointer',
-  },
-  subMenuItem: {
-    paddingTop: 15,
-    paddingBottom: 15,
-    paddingLeft: 10,
+  })
+
+  export const subMenuItem = mq({
+    paddingTop: '15px',
+    paddingBottom: '15px',
+    paddingLeft: 15,
     paddingRight: 10,
-    width: 160,
-  },
-  subMenuItemLeft: {
-    paddingTop: 15,
-    paddingBottom: 15,
-    paddingLeft: 10,
+    width: ['20vw','20vw', 160],
+  })
+
+  export const subMenuItemLeft = mq({
+    paddingTop: '15px',
+    paddingBottom: '15px',
+    paddingLeft: 15,
     paddingRight: 10,
-    width: 160,
+    width: ['20vw','20vw', '160px!important'],
     textAlign: 'right',
-  },
-  brand: {
-    width: '25%',
-  },
-  menuContainer: {
+  })
+
+  export const brand = mq({
+    float: 'left',
+    minWidth: ['100%!important','100%!important','100%!important','25%!important'],
+  })
+
+  export const menuContainer = mq({
+    display: 'block',
     transition : 'top 0.5 ease-in',
     backgroundColor: 'transparent!important',
     borderWidth: '0!important',
     marginBottom: '0!important',
     minHeight: '230px!important',
-    paddingLeft: 160,
-    paddingRight: 160,
+    paddingLeft: ['32px','32px','80px','140px','160px'],
+    paddingRight: ['32px','32px','80px','140px','160px'],
     paddingTop: 20,
-  },
-}
+  })
+
+  export const dropDownWrap = mq({
+    paddingLeft: ['0px','0px','0px','0px'], 
+    paddingTop: ['40px','40','0','0'],
```

Comment:
@Alphascythian Same stuff as mentioned before, this is way too much, because the element inside the menu already as a `15px` padding top. Once again, please pay more attention to the details.